HACKING.md

```diff
@@ -27,6 +27,31 @@ If you have not done so already, follow the instructions in the
 and [Getting Started](https://github.com/openshift/origin/blob/v4.0.0-alpha.0/docs/cluster_up_down.md#getting-started)
 sections to set up your environment and start the Docker daemon.
 
+If on Fedora 29, one extra step is needed. Change the
+`/usr/lib/systemd/system/docker.service` unit file to use the
+`cgroupfs` cgroup driver instead of `systemd`.
+(See https://bugzilla.redhat.com/show_bug.cgi?id=1558425#c22
+for more info).
+
+Change this line (under `ExecStart`):
+
+```
+--exec-opt native.cgroupdriver=systemd \
+```
+
+to:
+
+```
+--exec-opt native.cgroupdriver=cgroupfs \
+```
```

Comment:
Note that on AH/Silverblue, this won't work. The more canonical way to do this is `systemctl edit <svc>` so it adds an override in `/etc`. Anyway, totally fine to do this as a follow-up!

Reply:
> this won't work.

specifically it won't work to edit the `/usr/lib/systemd/system/docker.service` file since it is on a read-only filesystem

Reply:
+1 for follow-up